docs/pages/enroll-resources/database-access/enroll-managed-databases/oracle-exadata.mdx

```diff
@@ -0,0 +1,399 @@
+---
+title: Database Access with Oracle Exadata
+description: How to configure Teleport database access with Oracle Exadata.
+---
+
+(!docs/pages/includes/database-access/self-hosted-introduction.mdx dbType="Oracle Exadata"!)
+
+## How it works
+
+(!docs/pages/includes/database-access/how-it-works/mtls.mdx db="Oracle"!)
+
+<Tabs>
+<TabItem scope={["enterprise"]} label="Teleport Enterprise (Self-Hosted)">
+![Teleport Database Access Self-hosted Oracle](../../../../img/database-access/guides/oracle_selfhosted.png)
+</TabItem>
+<TabItem scope={["cloud"]} label="Teleport Enterprise (Cloud)">
+![Teleport Database Access Oracle Cloud](../../../../img/database-access/guides/oracle_selfhosted_cloud.png)
+</TabItem>
+</Tabs>
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+- Oracle Exadata server instance 19c or later.
+- The `sqlcl` [Oracle client](https://www.oracle.com/pl/database/sqldeveloper/technologies/sqlcl/) installed and added to your system's `PATH` environment variable or any GUI client that supports JDBC
+  Oracle thin client.
+- (!docs/pages/includes/tctl.mdx!)
+
+
+## Step 1/6. Configure Oracle Exadata
+
+<Admonition type="note">
+
+This guide assumes default configuration of the Oracle Exadata system. In particular:
+- Preconfigured TCPS listener.
+- An existing database certificate in the grid wallet.
+- A single Oracle Exadata VM is accessible to the `opc` user through SSH with a hostname `<Var name="demodb-vm" />`. Repeat the steps to configure additional VMs.
+- The SCAN DNS name is `<Var name="demodb-vm-scan.exadatadomain.oke.oraclevcn.com" />`.
+
+Adjust the commands to match your configuration.
+
+</Admonition>
+
+### Update per-database configuration
+
+Connect to the Oracle Exadata VM by logging in as the `opc` user and then switch to the `oracle` user:
+
+```bash
+$ ssh opc@<Var name="demodb-vm" />
+$ sudo su - oracle
+```
+
+For each database-specific Oracle home, update the `$ORACLE_HOME/network/admin/sqlnet.ora` file with the following entries:
+
+```
+SSL_CLIENT_AUTHENTICATION = TRUE
+SQLNET.AUTHENTICATION_SERVICES = (ALL)
+```
+
+For example, if there is a database named `spark`, the `oracle` user should have access to the automatically generated `/home/oracle/spark.env` file, which contains database-specific environment variables. After sourcing this file, the path `$ORACLE_HOME/network/admin/sqlnet.ora` will point to the `sqlnet.ora` file for the `spark` database and its associated Oracle home.
+
+```code
+# Load environment variables for the 'spark' database
+$ . spark.env 
+
+# Edit the configuration file at $ORACLE_HOME/network/admin/sqlnet.ora
+...
```

Comment:
Just to double check ignore if that is not correct. 

When I have playing with Oracle Gird I have been using: 

`. oraenv` 

that allow to select arbitrary SID 

Does the . spark.env configuration way will work with multiple spark_PDB* ?

Reply:
There was no `oraenv` file for Oracle Exadata, just a single `<foo>.env` file for each attached database. Pretty sure you can just override the variables yourself to whatever values you prefer, however. You don't _need_ the env files, you could provide these values yourself if you wanted to.